nearcore/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nearcore"
-version = "0.0.0"
+version = "1.23.0"
```

Comment:
we never updated this version in the past .. is it needed now? or should we leave it as 0.0. ?

Reply:
This is part of release commit for 1.23, it shows with my changes just because I started with that tag

Reply:
@bowenwang1996 This was meant for testing, tbh. To merge this to master would require dealing with version issues, and I, of course, would rebase it (or create separate PR)